test/molecule-role/molecule/compose/prepare.yml

```diff
@@ -13,12 +13,14 @@
       copy:
         src: "files/{{ item }}"
         dest: "/home/ubuntu/{{ item }}"
+        mode: preserve
       with_items:
         - docker-compose.yml
+        - create-topics.sh
     - name: Gather facts (we need the docker network interface ip)
       setup:
     - name: Run Docker compose
-      command: docker-compose up --scale stackstate-books-app=2 --scale stackstate-authors-app=2 -d
+      command: docker-compose up -d
```

Comment:
don't we test 2 instance of the apps anymore ?

Reply:
We bumped the memory of the apps when it was still using hibernate, but yes in this test case it actually doesn't make sense to have 2 instances. We are only sending one request so we'll always just have 1 service each

Reply:
I see your point